README.md

```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
 
 ### Generating measures-data.json
 To regenerate the `measures-data.json` file, which contains additional metadata and conforms to
-the measures schema, run:
+the measures schema, do the following:
+
+1. download the qpp quality pdfs 
+	```
+	wget https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip .
```

Comment:
For those without wget (I think that's a linux thing, right?)
`curl -o ./QPP_quality_measure_specifications.zip https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip`